source/docs/software/examples-tutorials/drivesim-tutorial/drivetrain-model.rst

```diff
@@ -0,0 +1,132 @@
+Step 2: Creating a Drivetrain Model
+===================================
+In order to accurately determine how your physical drivetrain will respond to given motor voltage inputs, an accurate model of your drivetrain must be created. This model is usually created by measuring various physical parameters of your real robot. In WPILib, this drivetrain simulation model is represented by the ``DifferentialDrivetrainSim`` class.
+
+Creating a ``DifferentialDrivetrainSim`` from Physical Measurements
+-------------------------------------------------------------------
+One way to creating a ``DifferentialDrivetrainSim`` instance is by using physical measurements of the drivetrain and robot -- either obtained through CAD software or real-world measurements. This constructor takes the following parameters:
+
+ * The type and number of motors on one side of the drivetrain.
+ * The gearing between the motors and the wheels as output over input (this number is usually greater than 1 for drivetrains).
+ * The moment of inertia of the drivetrain (this can be obtained from a CAD model of your drivetrain. Usually, this is between 3 and 8 kg m^2).
```

Comment:
```suggestion
 * The moment of inertia of the drivetrain (this can be obtained from a CAD model of your drivetrain. Usually, this is between 3 and 8 kg :math:`m^2`).
```